app/src/main/java/fr/groggy/racecontrol/tv/ui/home/HomeItemPresenter.kt

```diff
@@ -0,0 +1,31 @@
+package fr.groggy.racecontrol.tv.ui.home
+
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.leanback.widget.Presenter
+import fr.groggy.racecontrol.tv.R
+
+class HomeItemPresenter: Presenter() {
+    companion object {
+        private val TAG = HomeItemPresenter::class.simpleName
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup): ViewHolder {
+        Log.d(TAG, "onCreateViewHolder")
+        val view = LayoutInflater.from(parent.context)
+            .inflate(R.layout.home_item, parent, false)
+        return ViewHolder(view)
+    }
+
+    override fun onBindViewHolder(viewHolder: ViewHolder?, item: Any?) {
```

Comment:
I think the viewHolder is never nullable in this case... if you remove the `?` from the parameter does it work?

Reply:
Yeah, it did work :)